.test-infra/jenkins/job_PostCommit_Python_Chicago_Taxi_Example_Dataflow.groovy

```diff
@@ -19,46 +19,44 @@
 import CommonJobProperties as commonJobProperties
 import PostcommitJobBuilder
 import CronJobBuilder
+import LoadTestsBuilder
 
-
-// This job runs the Chicago Taxi Example script on Dataflow
-PostcommitJobBuilder.postCommitJob(
-    'beam_PostCommit_Python_Chicago_Taxi_Dataflow',
-    'Run Chicago Taxi on Dataflow',
-    'Google Cloud Dataflow Runner Chicago Taxi Example',
-    this
-) {
-    description('Runs the Chicago Taxi Example on the Dataflow runner.')
+def chicagoTaxiJob = { scope ->
+    scope.description('Runs the Chicago Taxi Example on the Dataflow runner.')
 
     // Set common parameters.
-    commonJobProperties.setTopLevelMainJobProperties(delegate)
+    commonJobProperties.setTopLevelMainJobProperties(scope)
+
+    def pipelineOptions = [
+        num_workers          : 5,
+        autoscaling_algorithm: 'NONE',
+    ]
 
     // Gradle goals for this job.
-    steps {
+    scope.steps {
         gradle {
             rootBuildScriptDir(commonJobProperties.checkoutDir)
+            commonJobProperties.setGradleSwitches(delegate)
             tasks(':sdks:python:test-suites:dataflow:py2:chicagoTaxiExample')
             switches('-PgcsRoot=gs://temp-storage-for-perf-tests/chicago-taxi')
+            switches("-PpipelineOptions=\"${LoadTestsBuilder.parseOptions(pipelineOptions)}\"")
         }
     }
 }
 
+PostcommitJobBuilder.postCommitJob(
+    'beam_PostCommit_Python_Chicago_Taxi_Dataflow',
+    'Run Chicago Taxi on Dataflow',
+    'Google Cloud Dataflow Runner Chicago Taxi Example',
+    this
+) {
+    chicagoTaxiJob(delegate)
+}
+
 CronJobBuilder.cronJob(
```

Comment:
With CronJobBuilder and PostcommitJobBuilder, do we run these test as frequent as specified by Cron config or more frequently (as often as new commits come in) ?

Reply:
As frequent as specified by Cron only. 

https://github.com/apache/beam/blob/0fc74c09f1b90c1b954b78973e83483daa51fd18/.test-infra/jenkins/CronJobBuilder.groovy#L47

The fourth parameter, `triggerOnCommit`, is set to False here, which means that the job is not triggered when new commits come in.